src/WorkItemFiling/AzureDevOpsClientWrapper.cs

```diff
@@ -64,31 +64,31 @@ public override async Task<IEnumerable<WorkItemModel>> FileWorkItems(IEnumerable
                     {
                         Operation = Operation.Add,
                         Path = $"/fields/{AzureDevOpsFieldNames.Title}",
-                        Value = metadata.Title
+                        Value = workItemModel.Title
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
                         Path = $"/fields/{AzureDevOpsFieldNames.ReproSteps}",
-                        Value = metadata.Description
+                        Value = workItemModel.Description
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
                         Path = $"/fields/{AzureDevOpsFieldNames.AreaPath}",
-                        Value = metadata.Area
+                        Value = workItemModel.Area
                     },
                     new JsonPatchOperation
                     {
                         Operation = Operation.Add,
                         Path = $"/fields/{AzureDevOpsFieldNames.Tags}",
-                        Value = string.Join(",", metadata.Tags)
+                        Value = string.Join(",", workItemModel.Tags)
                     }
                 };
 
-                if (metadata.CustomFields != null)
+                if (workItemModel.CustomFields != null)
                 {
-                    foreach (var customField in metadata.CustomFields)
+                    foreach (var customField in workItemModel.CustomFields)
```

Comment:
>var [](start = 29, length = 3)

Replace with type here

Reply:
Yes

---
In reply to: [377348570](https://github.com/microsoft/sarif-sdk/pull/1749#discussion_r377348570) [](ancestors = 377348570)